press/press/doctype/release_group/release_group.json

```diff
@@ -302,28 +299,10 @@
   {
    "fieldname": "column_break_njfg",
    "fieldtype": "Column Break"
-  },
-  {
-   "fieldname": "gunicorn_config_section",
-   "fieldtype": "Section Break",
-   "label": "Gunicorn Config"
-  },
-  {
-   "default": "sync",
-   "fieldname": "gunicorn_worker_type",
-   "fieldtype": "Select",
-   "label": "Gunicorn Worker Type",
-   "options": "\nsync\ngthread"
-  },
-  {
-   "description": "Setting this to non-zero value will convert sync to gthread worker. This doesn't apply to gevent. ",
-   "fieldname": "gunicorn_threads_per_worker",
```

Comment:
Umm, we still need this on release group... We can't configure individual benches for experimentation. 

I needed it so I can enable it on internal benches permanently and eventually some public benches.

Reply:
https://github.com/frappe/press/pull/1278